avocado/utils/iso9660.py

```diff
@@ -301,7 +302,7 @@ def __init__(self, path):
         super(Iso9660Mount, self).__init__(path)
         self._mnt_dir = tempfile.mkdtemp(prefix='avocado_' + __name__)
         process.run('mount -t iso9660 -v -o loop,ro %s %s' %
-                    (path, self.mnt_dir))
+                    (path, self.mnt_dir), sudo=True)
```

Comment:
Please read the commit messages. There was no sudo support at all in this class. So first I fixed the API, then I focused on adding sudo support. I don't think I should change the order.